misc/services/camel/de-metas-camel-externalsystems/de-metas-camel-shopware6/src/main/java/de/metas/camel/externalsystems/shopware6/api/model/order/OrderCandidate.java

```diff
@@ -49,16 +48,11 @@
 	@JsonProperty("salesRepId")
 	String salesRepId;
 
-	@NonNull
+	@Nullable
 	public String getCustomField(@NonNull final String customPath)
 	{
 		final JsonNode customIdNode = orderNode.at(customPath);
 
-		if (customIdNode == null || Check.isBlank(customIdNode.asText()))
-		{
-			throw new RuntimeException("Failed to process order " + jsonOrder.getOrderNumber() + " (ID=" + jsonOrder.getId() + "); Nothing found on the given customPath: " + customPath);
-		}
-
-		return customIdNode.asText();
+		return (customIdNode == null) ? null : customIdNode.asText();
```

Comment:
```suggestion
		return (customIdNode == null || customIdNode.isNull()) ? null : customIdNode.asText();
```

if the property was there but with value `null` then afaiu we would otherwise return the string `"null"`